pages/common/pabcnetcclear.md

```diff
@@ -0,0 +1,12 @@
+# pabcnetcclear
+
+> Preprocess and compile PascalABC.NET source files.
+> More information: <http://pascalabc.net/en/>.
+
+- Compile source file into executable:
+
+`pabcnetcclear {{source.pas}}`
+
+- Compile source file into executable along with debug info:
```

Comment:
Does it save the debug info to a file or print it?
```suggestion
- Compile a source file into an executable along with debug info:
```

Reply:
It only saves debug info into .mdb (on Linux).